lib/bike_brigade/delivery.ex

```diff
@@ -1012,4 +1056,19 @@ defmodule BikeBrigade.Delivery do
   def change_opportunity(%Opportunity{} = opportunity, attrs \\ %{}) do
     Opportunity.changeset(opportunity, attrs)
   end
+
+  @doc """
+  List all task assignment logs
+
+  Currently this table is used for analytics and not exposed in the app.
+  """
+  def list_task_assignment_logs() do
+    Repo.all(TaskAssignmentLog)
+  end
+
+  def create_task_assignment_log(attrs \\ %{}) do
+    %TaskAssignmentLog{timestamp: DateTime.utc_now()}
+    |> TaskAssignmentLog.changeset(attrs)
+    |> Repo.insert()
+  end
```

Comment:
Nit: do you think this might belong in a new context, "tracking" or named something similar?

Reply:
@teesloane what do you mean by on the main PR?

I actually think that putting this in a new context makes sense!

Reply:
When I wrote this comment, I wrote it while going through each commit one by one; I figured it might get lost when submitting my review; I meant to write the comment on the page for the PR as a whole, if that makes sense (typing this now, it doesn't make sense, but now typing that it doesn't make sense, — well, it's all starting to make sense.)

He typed from a chair as the wind battered at the trees outside his window. It was getting late and Ty was thinking about going to bed.
xo